index.bs

```diff
@@ -519,10 +519,14 @@ Individuals Representing a Member Organization</h5>
 	the number of individuals representing a W3C Member
 	(or group of <a href="#MemberRelated">related Members</a>).
 
-<h4 id="GeneralMeetings">
+<h4 id="GeneralMeetings" oldids="GAEvents, EventsS">
 Meetings</h4>
 
-	The requirements in this section apply to the official meetings of any [=W3C group=].
+	The requirements in this section apply to the official meetings of any [=W3C group=],
+	as well as to official W3C meetings with open-ended participation,
+	such as <dfn id="EventsW" lt="Workshop | W3C Workshop" export>Workshops</dfn>
+	(as described in the <a href="https://www.w3.org/Guide/meetings/workshops">Guide article about Workshops</a> [[GUIDE]])
+	or other events open to the whole [=Member|Membership=] or to the public.
```

Comment:
Can we shorten this to

```suggestion
	such as <a id="EventsW" href="https://www.w3.org/Guide/meetings/workshops">Workshops</a>. [[GUIDE]]
```

?

Reply:
Maybe with s/open-ended participation/open-ended participation from the Membership and/or the public/ if we want to be clear.

Reply:
addressed by the rewrite done after last meeting